src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -142,6 +153,14 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
                     if (c.AccountId != oldContacts[i].AccountId) {
                         if (c.AccountId != null) {
                             listAccountIdHHToUpdate.add(c.AccountId);
+                            if (c.Account.RecordTypeId == UTIL_Describe.getHhAccRecTypeID()
+                                    && UTIL_CustomSettingsFacade.getSettings().Automatic_Household_Naming__c != false){
+                                setAccountIdHHToRename.add(c.AccountId);
+                            }
```

Comment:
I think in here, we also need to update oldContacts[i].Account name if oldContacts[i].AccountId != null.

Reply:
Good catch!